### PR TITLE
Journal/Tutorial Week 4

### PR DESCRIPTION
**Pull request by ericksonk**

### CODE REVIEW

**Comment by jeffworley** [comment]
Excellent topic choices for both journal entries!!!  I am looking forward to the wisdom you will impart regarding successful use of shortcuts in package.json files as well as your examples for semantical html. :+1:

**Comment by nickycadavillo** [comment]
:+1: So far your journal was a great read; love the '97 chessboard reference. Haha!  The tutorial will definitely be helpful in the future.  Looking forward to seeing the finished tutorial!

**Comment by ShayHall** [comment]
:+1: Good start.  I think semantic HTML will be a good topic.

**Comment by jeremyrist** [comment]
#profit :+1:

File: journal-week-4.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Week 4 Overview
+
+* semantic html: describes the content of your web pages more accurately and allows other programs like screen readers and search engines to use the given info
+
+* sass- css processor
+  - refactored old css using sass
+  - `node-sass`
+
+* chess
+  - built chessboard that would have been cool circa 1997
```
Comment (kellymurray) [inline journal-week-4.md]: :+1:

**Pull request merged**